viewing_party/party.py

```diff
@@ -1,23 +1,155 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        movie_dict = { 
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return movie_dict
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    watchlist = user_data.get("watchlist", [])
+    watched = user_data.get("watched", [])
+
+    found_movie = None
+
+    for movie in watchlist:
+        if movie["title"] == title:
+            found_movie = movie
+            watchlist.remove(movie)
+            watched.append(movie)

```

Comment:
Nitpick: I like how this reads since `watched = user_data.get("watched", [])` etc, but this is also a good opportunity to use the previous functions:

```suggestion
            add_to_watched(user_data, movie)
```